test/e2e/data/e2e_conf.yaml

```diff
@@ -42,7 +42,7 @@ providers:
         value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.1/core-components.yaml"
         type: "url"
         files:
-          - sourcePath: "./shared/v1alpha4/metadata.yaml"
+          - sourcePath: "./shared/v1beta1/metadata.yaml"
```

Comment:
This file doesn't exist. Needs to be copied from https://github.com/kubernetes-sigs/cluster-api/blob/main/metadata.yaml